docs: add deployment info for Qovery (#1352)
nuxt/nuxtjs.org: content/en/guides/deployment/deployment-qovery.md

---
title: Deploy Nuxt on Qovery
description: How to deploy Nuxt.js on Qovery?
menu: Qovery

> target: Server

Reviewer: should this be Static & Server?
```suggestion
target: Static & Server
```

category: deployment
position: 109
---

[Qovery](https://qovery.com) is a fully-managed cloud platform that runs on your AWS, GCP, Azure and Digital Ocean account where you can host static sites, backend APIs, databases, cron jobs, and all your other apps in one place.

Static sites are **completely free** on Qovery and include the following:

- Continuous, automatic builds & deploys from GitHub, Bitbucket, and GitLab.
- Automatic SSL certificates through [Let's Encrypt](https://letsencrypt.org).
- Free managed PostgreSQL.
- Free SSD storage.
- Unlimited collaborators.
- Unlimited [custom domains](https://docs.qovery.com/guides/getting-started/setting-custom-domain/).

## Prerequisites

> This guide assumes you already have a Nuxt project to deploy. If you need a project, use the [Get Started](/content/en/get-started/installation.md).

Reviewer: ```suggestion
This guide assumes you already have a Nuxt project to deploy. If you need a project, follow the [Get Started](/content/en/get-started/installation.md) guide.
```

> This guide assumes you already have a Nuxt project to deploy. If you need a project, follow the [Get Started](/content/en/get-started/installation.md) guide.

Reviewer: ```suggestion
This guide assumes you already have a Nuxt project to deploy. If you need a project, follow the [Get Started](/docs/2.x/get-started/installation) guide.
```

## Setup

> Follow the procedure below to set up a Nuxt on Qovery:

Reviewer: ```suggestion
Follow the procedure below to set up Nuxt on Qovery:
```
